### ISSUE TITLE
interferometric phase image

### ISSUE THREAD

**Issue by keepfighting666** [mdelgadoblasco/snap2stamps#26]
i open one phase image in snap，and it looks strange in red box
is something wrong in data process ？
![image](https://github.com/user-attachments/assets/a9383fff-1f11-4975-9554-6b7cbc5fc44d)

**Comment by mdelgadoblasco** [comment]
No, it is something inherent to the SNAP processing, most probably related to the extent of the DEM tiles
